examples/Aggregator/Server/Server.csproj

```diff
@@ -10,6 +10,19 @@
     <Protobuf Include="..\Proto\count.proto" GrpcServices="Both" Link="Protos\count.proto" />
 
     <PackageReference Include="Grpc.AspNetCore" Version="$(GrpcDotNetPackageVersion)" />
+
+    <PackageReference Include="OpenTelemetry" Version="0.2.0-alpha.51" />
```

Comment:
Use dependencies.props to manage these? Also, I would still use the same location for managing feeds as well.